nilearn/connectome/__init__.py

```diff
@@ -5,7 +5,8 @@
 """
 
 from .connectivity_matrices import (sym_matrix_to_vec, vec_to_sym_matrix,
-                                    ConnectivityMeasure)
+                                    ConnectivityMeasure, cov_to_corr,
```

Comment:
Yes I rebased but when using mergetool to fix conflicts I skipped accidentally this one